.pre-commit-config.yaml

```diff
@@ -0,0 +1,18 @@
+default_language_version:
+  python: python3
+repos:
+  - repo: local
+    hooks:
+      - id: prospector
```

Comment:
Prospector is a "local" hook, so it'll use the local virtualenv. By default, every repo gets its own virtualenv, which means it doesn't have access to the other dependencies. That's usually fine, but prospector runs pylint, which needs access to the whole build project